app/src/main/res/values-sw600dp-port/dimens.xml

```diff
@@ -467,6 +467,23 @@
   <dimen name="hints_and_solution_recycler_view_padding_top">0dp</dimen>
   <dimen name="hints_and_solution_recycler_view_padding_bottom">112dp</dimen>
 
+  <!-- Hints summary  -->
+  <dimen name="hints_summary_constraint_layout_layout_margin_start">128dp</dimen>
+  <dimen name="hints_summary_constraint_layout_layout_margin_end">128dp</dimen>
+  <dimen name="hints_summary_constraint_layout_layout_margin_bottom">4dp</dimen>
+
+  <dimen name="hint_list_container_layout_margin_start">128dp</dimen>
+  <dimen name="hint_list_container_layout_margin_end">128dp</dimen>
+  <dimen name="hint_list_container_layout_margin_bottom">32dp</dimen>
+
+  <dimen name="hint_title_margin_top">44dp</dimen>
+  <dimen name="hint_title_padding_bottom">28dp</dimen>
+
+  <dimen name="reveal_hint_btn_margin_bottom">28dp</dimen>
+
+  <dimen name="expand_list_icon_margin_top">32dp</dimen>
```

Comment:
To maintain consistency, make sure that all dimensions are present in all four dimens.xml files. For example, here you mentioned just two dimensions for `expand_list_icon*`. So, add another two dimensions i.e. `hints_summary_expand_list_icon_margin_end` and `hints_summary_expand_list_icon_margin_start` and assign 0 dp. Do these for other dimensions also